sunpy/map/mapbase.py

```diff
@@ -2762,17 +2763,15 @@ def plot(self, *, annotate=True, axes=None, title=True, autoalign=False,
 
         Notes
         -----
-        The ``autoalign`` functionality is computationally intensive. If the plot will
+        The ``autoalign`` functionality can be intensive to render. If the plot is to
         be interactive, the alternative approach of preprocessing the map (e.g.,
         de-rotating it) to match the desired axes will result in better performance.
 
         When combining ``autoalign`` functionality with
         `~sunpy.coordinates.Helioprojective` coordinates, portions of the map that are
-        beyond the solar disk may not appear, which may also inhibit Matplotlib's
-        autoscaling of the plot limits. The plot limits can be set manually.
-        To preserve the off-disk parts of the map, using the
-        `~sunpy.coordinates.SphericalScreen` context
-        manager may be appropriate.
+        beyond the solar disk may not appear.  To preserve the off-disk parts of the
+        map, using the `~sunpy.coordinates.SphericalScreen` context manager may be
+        appropriate.
         """
         # Todo: remove this when deprecate_positional_args_since is removed
```

Comment:
We can pull this for 7.0, don't have to do it now, just saw it.

Reply:
Removed in #8165

Reply:
This PR was not rebased with that in?